package.json

```diff
@@ -13,22 +13,25 @@
     "url": "git+https://github.com/solidjs/solid-jest.git"
   },
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "test:integration": "./run-integration.sh"
   },
   "bugs": {
     "url": "https://github.com/solidjs/solid/issues"
   },
   "devDependencies": {
     "babel-preset-solid": "^1.0.0",
-    "jest": "^27.0.0",
+    "jest": "^29.0.0",
     "solid-js": "^1.0.0"
   },
   "peerDependencies": {
-    "babel-preset-solid": "^1.0.0"
+    "babel-preset-solid": "^1.0.0",
+    "jest": "^29.0.0"
```

Comment:
I added jest as a peer dependency so you get a warning if you have a version mismatch.